src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -198,6 +198,14 @@ START_SECTION(integratePeak())
   TEST_REAL_SIMILAR(peak_height, 966489.0)
   TEST_REAL_SIMILAR(peak_apex_pos, 2.7045)
 
+  right = 3.011416667;
+  ptr->setIntegrationType("simpson");
+  ptr->integratePeak(chromatogram, left, right, peak_area, peak_height, peak_apex_pos);
+  cout << "simpson (even number of points): " << endl;
+  TEST_REAL_SIMILAR(peak_area, 71720.443144994)
```

Comment:
It does look like you are getting the same value.  I think I maybe misunderstanding something then.  It also does not appear to "underestimate" the area as it is less than that computed by the trapezoid rule.